src/sonic-yang-models/doc/Configuration.md

```diff
@@ -922,6 +922,9 @@ instance is supported in SONiC.
     },
     "QUEUE": {
         "FLEX_COUNTER_STATUS": "enable"
+    },
+    "TUNNEL": {
+        "FLEX_COUNTER_STATUS": "enable"

```

Comment:
>FLEX_COUNTER_STATUS [](http://example.com/codeflow?start=9&length=19)

POLL_INTERVAL is missing on 4 entries. Is it expected?

Reply:
I have added examples

Reply:
Could you replace tab with spaces?

Reply:
Done